cairo-sys-rs/build.rs

```diff
@@ -70,3 +72,19 @@ fn find(package_name: &str, shared_libs: &[&str]) -> Result<(), Error> {
         Err(err) => Err(err),
     }
 }
+
+// HACK: if libffi is installed via homebrew it doesn't get linked correctly.
+// this adds the homebrew libffi to the pkg-config-path.
+fn macos_maybe_find_lib_ffi() {
+    #[cfg(target_os = "macos")]
+    {
+        let brew_prefix = process::Command::new("brew")
+            .arg("--prefix")
+            .output()
+            .map(|o| PathBuf::from(String::from_utf8_lossy(&o.stdout).trim()))
+            .unwrap_or(PathBuf::from("/usr/local"));
+        let libffi_path = brew_prefix.join("opt/libffi/lib/pkgconfig");
+        env::set_var("PKG_CONFIG_PATH", libffi_path);
```

Comment:
If the `PKG_CONFIG_PATH` variable is already set, you're completely overwriting it... Not very ideal. :-/

Reply:
should I infer from this review that you would consider taking this patch if it were less bad?

Reply:
Honestly yes. It's been an annoying issue for far too long already. I never did it myself because I felt like it seemed way too much but I wasn't able to come up with a solution in the meantime, so...